File: sdkwork-ai-sdk-csharp/Models/ContextMessage.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class ContextMessage
    {
        public string? Role { get; set; }
        public string? Content { get; set; }
        public string? Name { get; set; }
        public List<ToolCall>? ToolCalls { get; set; }
    }
}

File: sdkwork-ai-sdk-typescript/src/types/context-message.ts
import type { ToolCall } from './tool-call';

/** 上下文消息 */
export interface ContextMessage {
  /** 角色: system, user, assistant */
  role?: string;
  /** 消息内容 */
  content?: string;
  /** 发送者名称 */
  name?: string;
  /** 工具调用信息 */
  toolCalls?: ToolCall[];
}
